docs/reproducing.md

```diff
@@ -56,6 +65,14 @@ $ python infra/helper.py reproduce $PROJECT_NAME <fuzz_target_name> <testcase_pa
 ```
 
   This is essentially the previous command that additionally mounts local sources into the running container.
+
+- *Reproduce bad build error:*
```

Comment:
Move this to its own big section at end like "# Reproducing OSS-Fuzz issues"
as
"# Reproducing OSS-Fuzz bad build failures". 

this is not the right place, this content is for oss-fuzz issues in issue tracker. Also, would need to repeat the pull docker image section.

Reply:
Done. Should we move "pull_images" into a separate section as well?